helix-core/src/main/java/org/apache/helix/HelixRebalanceException.java

```diff
@@ -27,6 +27,7 @@
     INVALID_CLUSTER_STATUS,
     INVALID_REBALANCER_STATUS,
     FAILED_TO_CALCULATE,
+    INVALID_INPUT,
```

Comment:
I agree. This shall be done when we finish the error reporting feature. The description will be fully used when it is reported in either log or some alerts.

Reply:
@jiajunwang Could you please put a TODO with what you said in your comment?

Reply:
I do not see a TODO...

Reply:
Sorry that I forgot to update the comments here. On my second thought, I don't think we need description. Because the error message shall contain enough information.
The type here is more for monitoring purpose.

But we may need to add some static fields in the enum instances for the monitor. I will add this TODO.
Thanks for reminding me.